frontend/javascripts/oxalis/model/bucket_data_handling/texture_bucket_manager.js

```diff
@@ -388,9 +388,16 @@ export default class TextureBucketManager {
         for (const baseBucketAddress of baseBucketAddresses) {
           const lookUpIdx = this._getBucketIndex(baseBucketAddress);
           const posInBuffer = channelCountForLookupBuffer * lookUpIdx;
-          if (this.lookUpBuffer[posInBuffer] !== -2 || lookUpIdx === -1) {
-            // Either, another bucket was already placed here. Or, the lookUpIdx is
-            // invalid. Skip the entire loop
+          if (lookUpIdx === -1) {
+            // The lookUpIdx is invalid. Skip the entire loop.
+            break;
+          } else if (
+            this.lookUpBuffer[posInBuffer] !== -2 &&
+            this.lookUpBuffer[posInBuffer + 1] < bucketZoomStep
+          ) {
+            // Another bucket was already placed here with a better zoomstep.
+            // Skip the entire loop.
+            // TODO: clarify flicker when zooming from mag 4 to mag 8.
```

Comment:
todo: I might need a rubber duck here :see_no_evil:

Reply:
I reverted the changes and extracted them to a separate PR: https://github.com/scalableminds/webknossos/pull/6029. Since this proved to be a bit tricky, a separate PR is easier to revert if it turns out to cause trouble in production. Also, it doesn't block the current PR then.